src/index.html

```diff
@@ -1,14 +1,40 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+
+    <title>Moyo header</title>
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a class="logo" href="index.html">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li><a class="nav-link is-active" href="index.html">Apple</a></li>
+          <li><a class="nav-link" href="index.html">Samsung</a></li>
+          <li><a class="nav-link" href="index.html">Smartphones</a></li>
+          <li><a class="nav-link" href="index.html">Laptops & Computers</a></li>
```

Comment:
```suggestion
          <li>
            <a class="nav-link" href="index.html">
              Laptops & Computers
            </a>
          </li>
```

Check all other similar cases.